UndertaleModTool/Editors/UndertaleRoomEditor.xaml.cs

```diff
@@ -1449,7 +1492,7 @@ private void MoveItem(UndertaleObject obj, int dist, bool focus = true)
             };
             if (list is null)
             {
-                Debug.WriteLine($"Can't change object position - list of selected object not found.");
+                mainWindow.ShowError("Can't change the object position - a list for the selected object is not found.");
```

Comment:
```suggestion
                mainWindow.ShowError("Can't change the object's position - no list for the selected object was found.");
```

Reply:
The "'s" shouldn't be used with inanimate objects, so it should be "Can't change a position of the object"

Reply:
Why? I'm not aware of such a rule in english